styleguide.config.js

```diff
@@ -88,7 +88,7 @@ module.exports = {
       ],
     },
     {
-      name: '🎨 Form',
+      name: 'Form',
```

Comment:
I relunctantly admit this is a good call. It's cute but there are even accessibility issues using emojis in important text so free-form (without the right markup). Plus it makes linking weird.

Semi-related: we'll need an `<Emoji>` component! 😆

Reply:
Yeah, it looked super garbagey too. Once we have an icon component we can implement icons for these sections that work better stylistically, but I think the emojis, whilst cute, weren't doing us any favours in terms of either aesthetics or accessibility. (And you're right, a screen reader would read these out, which is totally unnecessary!)